src/js/config/config.js

```diff
@@ -1,7 +1,7 @@
 var config = {
   // @@ENV gets replaced by build system
   environment: "@@ENV",
-  rootUrl: "",
+  rootUrl: "assets/",
```

Comment:
I would prefer production as default too, as it is less error-prone (i.e. you have to explicitly act to change the value to something that does not work in production)